source/core/output_buffers/ut_output_table_buffer.tpb

```diff
@@ -134,22 +139,27 @@ create or replace type body ut_output_table_buffer is
     l_lines sys_refcursor;
   begin
     open l_lines for
-      select column_value as text
+      select text, item_type
         from table(self.get_lines(a_initial_timeout, a_timeout_sec));
     return l_lines;
   end;
 
   overriding member procedure lines_to_dbms_output(self in ut_output_table_buffer, a_initial_timeout natural := null, a_timeout_sec natural := null) is
-    l_lines sys_refcursor;
-    l_line varchar2(32767);
+    l_data      sys_refcursor;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this weak REF CURSOR by one with a return type. ([plsql:WeakRefCursorCheck](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3AWeakRefCursorCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=809&issues=AWgA5Io6_MPaBDn0kbL3&open=AWgA5Io6_MPaBDn0kbL3)

[ISSUE_KEY]: AWgA5Io6_MPaBDn0kbL3